nicegui/client.py

```diff
@@ -73,6 +73,7 @@ def __init__(self, page: page, *, shared: bool = False) -> None:
         self._body_html = ''
 
         self.page = page
+        self.state = {}
```

Comment:
I think this should be an OberservalDict to allow users to add callbacks for when the data changes. And it would be consistent with the other storage options. Or is that not necessary because we only store in memory and people can use bindings?

Reply:
Updated. As the automatic content update of bindable properties worked perfectly I implicated NiceGUI had another, active approach here.